lib/database.js

```diff
@@ -7,7 +7,8 @@ class Database {
     } else if (program.type == "Azure" || "azure") {
       this._database = nodecloud.azure.compute(options);
     } else {
-      throw new Error("Please specify a provider by flag -p --provider");
+      const emoji = require("node-emoji");
```

Comment:
Do not put import statements here like this they should be done on the top.

Reply:
Since its a common thing you can move it to helper.js and import it as a function